vllm/entrypoints/openai/protocol.py

```diff
@@ -585,16 +685,65 @@ class FunctionCall(OpenAIBaseModel):
     name: str
     arguments: str
 
+    def to_dict(self):
+        return {"name": self.name, "arguments": self.arguments}
```

Comment:
why not just use pydantic dict? https://docs.pydantic.dev/1.10/usage/exporting_models/

Reply:
I found I could use model_dump and dict elsewhere and just forgot to remove this.